conans/client/migrations_settings.py

```diff
@@ -288,3 +288,75 @@
 settings_1_14_2 = settings_1_14_1
 settings_1_14_3 = settings_1_14_2
 settings_1_14_4 = settings_1_14_3
+
+settings_1_15_0 = """
```

Comment:
I think this might be better to be adjusted when the release is merged to develop probably. In that point, we have "freeze" the settings for the version, not before. Why did you add it?